_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -764,8 +815,14 @@ MFA enforced by IdP, not AWS::
 [[deployment_walkthrough]]
 == Deployment walkthrough
 
-Let's now walk through the step-by-step process of how to create a production-grade AWS account structure, fully
-defined and managed as code, using the Gruntwork Infrastructure as Code Library.
+Let's now walk through the step-by-step process of how to create a production-grade AWS account structure using the
+Gruntwork AWS Landing Zone solution. Doing this from scratch is time-consuming and complicated, and all the pre-built
+"Landing Zone" solutions we tried were expensive, low-quality, UI-driven, and/or didn't allow customization. The
+Gruntwork AWS Landing Zone solution allows us to do this in minutes, but it's also 100% defined as code, so you can
+fully customize it to your needs. The best part is that it automatically configures your accounts with a variety of
+authentication, access controls and security features by using AWS Organizations, IAM Roles, IAM Users, IAM Groups,
+IAM Password Policies, Amazon GuardyDuty, AWS CloudTrail, AWS Config, and a whole lot more so we can avoid doing
+this manually.
```

Comment:
You already introduced it in the Core Concepts section, so this is a 100% repeat. I'd reduce this back down to a single sentence:

> Let's now walk through the step-by-step process of how to create a production-grade AWS account structure using the Gruntwork AWS Landing Zone solution.

Reply:
Ok, roger.